src/constants/translations/en/index.js

```diff
@@ -48,7 +49,8 @@ const en = {
     completeProfile,
     tutorProfilePage,
     adminTable,
-    constant
+    constant,
+    appSelect
```

Comment:
i think it should be called filters where we could save translation for filters sort etc, or if your component have always different values , you should store it on page where it userd